src/timedate/timedated.c

```diff
@@ -916,6 +916,63 @@ static int method_set_ntp(sd_bus_message *m, void *userdata, sd_bus_error *error
         return sd_bus_reply_method_return(m, NULL);
 }
 
+static int method_list_timezones(sd_bus_message *m, void *userdata, sd_bus_error *error) {
+        sd_bus *bus = sd_bus_message_get_bus(m);
+        Context *c = userdata;
+        int interactive, r;
+        _cleanup_(sd_bus_message_unrefp) sd_bus_message *reply = NULL;
+        _cleanup_strv_free_ char **zones = NULL;
+        char** zone;
+
+        assert(m);
+        assert(bus);
+        assert(c);
+
+        r = sd_bus_message_read(m, "b", &interactive);
```

Comment:
this function should not take any argument. First of all, it shouldn't do polkit anyway, since it should not require authentication. Secondly, the explicit "interactive" boolean is unnecessary these days anyway, as the same info can be included in the bus message header these days, and is thus implicit. Hence, simplify this, drop the argument, and drop the polkit hookup